junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,64 @@
+
+from os.path import join
+from settings.common import ROOT_DIR
+import pandas as pd
+
+from django.shortcuts import get_object_or_404
+from django.core.exceptions import PermissionDenied
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from junction.conferences.models import Conference
+from junction.proposals.models import (
+    Proposal,
+    ProposalComment,
+    ProposalSectionReviewer,
+    ProposalSectionReviewerVoteValue,
+    )
+from junction.base.constants import (
+    ProposalReviewVote,
+    ProposalStatus,
+    ProposalReviewStatus,
+    )
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--conference_slug', default=None, help='Enter the conference slug where to export reviewer votes from')
+
+        parser.add_argument('--user_id', default=None, help='Enter your user id')
+
+    def handle(self, *args, **options):
+
+        conference = get_object_or_404(Conference, slug=self.options.get('conference_slug'))
+        user = User.objects.get(id=self.options.get('user_id'))
+
+        if not conference_is_moderator(user=user, conference=conference):
```

Comment:
@prasoonmayank  `conference_is_moderator ` is not defined anywhere in this file.

Reply:
True. I didn't run it after converting to csv, and think made some basic errors. Will make a note of such issues from now on.